### ISSUE TITLE
Inspect field .State.Health vs. .State.Healthcheck

### ISSUE THREAD

**Issue by white-gecko** [containers/podman#13720]
**Is this a BUG REPORT or FEATURE REQUEST? (leave only one on its own line)**

/kind bug

**Description**

In a script (currently for docker) with a command:

    docker inspect --format='{{json .State.Health.Status}}' container_name

however, for podman the field is called `Healthcheck`

    docker inspect --format='{{json .State.Healthcheck.Status}}'

To reach better convergence, I suggest to provide an alias for that.

**Steps to reproduce the issue:**

1. Get the output of `docker inspect` and `podman inspect`
2. Look for the field `Health` resp. `Healthcheck`

**Describe the results you received:**
For docker I can find `.State.Health.Status`
For podman I can find `.State.Healthcheck.Status`

**Describe the results you expected:**
I expect convergence to docker output, if this is possible

**Additional information you deem important (e.g. issue happens only occasionally):**

**Output of `podman version`:**

```
Version:      3.4.4
API Version:  3.4.4
Go Version:   go1.16.8
Built:        Wed Dec  8 22:45:07 2021
OS/Arch:      linux/amd64
```

**Output of `podman info --debug`:**

```
(paste your output here)
```

**Package info (e.g. output of `rpm -q podman` or `apt list podman`):**

```
podman-3.4.4-1.fc35.x86_64
```

**Have you tested with the latest version of Podman and have you checked the Podman Troubleshooting Guide? (https://github.com/containers/podman/blob/main/troubleshooting.md)**

No

**Additional environment details (AWS, VirtualBox, physical, etc.):**
local laptop

**Comment by mheon** [comment]
@flouthoc Mind taking this one?

**Comment by Luap99** [comment]
I am almost sure I fixed this at some point.

**Comment by white-gecko** [comment]
Sorry, I couldn't verify this with the latest version. So maybe you should verify it first :-\

**Comment by Luap99** [comment]
https://github.com/containers/podman/pull/11654, this is fixed in 4.0
